src/screens/Manager/AppsList/index.js

```diff
@@ -35,7 +35,7 @@ const AppsList = ({
   setStorageWarning,
   isInstalledView,
 }: Props) => {
-  const viewHeight = active ? "auto" : height - 267;
+  const viewHeight = active ? "auto" : height - 267 || 0;
```

Comment:
can you explain what `|| 0` is doing here?

Reply:
btw with precedence it's useless code. it evals as `active ? "auto" : height - (267 || 0);`

Reply:
i'm not sure as to how but while testing the main feature i got a random error on this viewHeight that gave a NaN instead of a number and broke the stylesheet. could not reproduce it easily so i just figured this could help :shrug:

Reply:
ho i should fix with parenthesis then